modules/video-recorder-junit/build.gradle

```diff
@@ -0,0 +1,28 @@
+plugins {
+  id 'java'
+}
+
+ext {
+  artifactId = "video-recorder-junit"
+}
+
+repositories {
+  mavenCentral()
+}
+
+dependencies {
+  api project(":modules:video-recorder-core")
+  testImplementation platform('org.junit:junit-bom:5.10.0')
+  testImplementation  'org.junit.jupiter:junit-jupiter'
+  implementation("org.junit.jupiter:junit-jupiter-api:5.10.2")
+  implementation("org.junit.jupiter:junit-jupiter-engine:5.10.2")
+  testImplementation("org.assertj:assertj-core:$assertjVersion") { transitive false }
+  implementation('org.bytedeco:javacv:1.5.10')
+  implementation("org.bytedeco:javacv-platform:1.5.10")
+}
```

Comment:
I would suggesting to apply variables into `gradle/dependencies.gradle` - in your choice naming
- `junit-bom`
- `junit-jupiter`
- `javacv`

And reuse it in this const
- example `testImplementation("org.assertj:assertj-core:$assertjVersion")`
```
dependencies {
  api project(":modules:video-recorder-core")
  testImplementation platform('org.junit:junit-bom:5.10.0')
  testImplementation  'org.junit.jupiter:junit-jupiter'
  implementation("org.junit.jupiter:junit-jupiter-api:5.10.2")
  implementation("org.junit.jupiter:junit-jupiter-engine:5.10.2")
  testImplementation("org.assertj:assertj-core:$assertjVersion") { transitive false }
  implementation('org.bytedeco:javacv:1.5.10')
  implementation("org.bytedeco:javacv-platform:1.5.10")
}
```

Also you can repeat this practice for
- `video-recorder-core`
- `video-recorder-junit`
- `video-recorder-testng`

Reply:
Good point. Thank you @ylazakovich